src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.application.name=LearningSystem
+spring.profiles.active=local
```

Comment:
`application.properties` is fine, but could you please switch to `yaml` format (just to see the difference later :) )